docs/abi-spec.rst

```diff
@@ -574,7 +574,9 @@ Constructor, receive, and fallback never have ``name`` or ``outputs``. Receive a
 
 An event description is a JSON object with fairly similar fields:
 
-- ``type``: always ``"event"``
+- ``type``: always ``"event"``:
+  * ``internalType``: ABI JSON's internalType describes a contract's internal function parameter or 
+           return value type, inaccessible outside the contract.
```

Comment:
There are several problems here:
- The `internalType` field is at the same level as `type`, not a part of it.
- Events actually don't have this field. Their arguments do.
- Not only events have arguments. If we list this field, we should do it consistently for objects that have them.
- Not sure if this will even render properly in the RST document. Especially the indent looks iffy.